### PR TITLE
[16.0][ADD] account_account_tag_code

### PR DESCRIPTION
**Pull request by MarinaAForgeFlow**

### CODE REVIEW

File: account_account_tag_code/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Copyright 2016-2017 ACSONE SA/NV
```
Comment (DavidJForgeFlow) [inline account_account_tag_code/__manifest__.py]: Change it to
# Copyright 2023 ForgeFlow S.L.

File: account_account_tag_code/views/account_account_tag_views.xml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        <field name="inherit_id" ref="account.account_tag_view_form" />
+        <field name="arch" type="xml">
+            <field name="name" position="after">
+                <field name="code" />
+            </field>
+        </field>
+    </record>
+
+    <record id="account_tag_view_tree" model="ir.ui.view">
+        <field name="model">account.account.tag</field>
```
Comment (DavidJForgeFlow) [inline account_account_tag_code/views/account_account_tag_views.xml:16]: Maybe using an specific name and id will improve the readability to other developers. Using a name like: "account.account.tag.tree - code" will help on this.

File: account_account_tag_code/views/account_account_tag_views.xml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="account_tag_view_form" model="ir.ui.view">
+        <field name="name">Tags</field>
```
Comment (DavidJForgeFlow) [inline account_account_tag_code/views/account_account_tag_views.xml]: Maybe using an specific name and id will improve the readability to other developers. Using a name like: "account.account.tag.form - code" will help on this.

File: account_account_tag_code/README.rst [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+Authors
+~~~~~~~
+
+* ForgeFlow
+
+Contributors
+~~~~~~~~~~~~
+
+* ForgeFlow Team.
```
Comment (DavidJForgeFlow) [inline account_account_tag_code/README.rst]: Usually we put the names of the developers here with the following format:
`* Marina Alapont <marina.alapont@forgeflow.com>`

**Review by DavidJForgeFlow — CHANGES_REQUESTED** [review]
Functional review :+1: I commented a few improvements to the code readability and format stuff. But great job!

File: account_account_tag_code/models/account_account_tag.py [hunk 5]
```diff
@@ -0,0 +1,7 @@
+from odoo import fields, models
+
+
+class AccountAccountTag(models.Model):
+    _inherit = "account.account.tag"
+
+    code = fields.Char(required=True, translate=False)
```
Comment (MiquelRForgeFlow) [inline account_account_tag_code/models/account_account_tag.py]:
```suggestion
    code = fields.Char(required=True)
```
char fields by default are translate=False

File: account_account_tag_code/__manifest__.py (hunk 1 above)
Comment (MarinaAForgeFlow) [reply account_account_tag_code/__manifest__.py]: Thanks, I used another module as a template and forgot to change it

File: account_account_tag_code/models/account_account_tag.py (hunk 5 above)
Comment (MarinaAForgeFlow) [reply account_account_tag_code/models/account_account_tag.py]: I wasn't sure, thanks.

**Review by DavidJForgeFlow — APPROVED** [review]
LGTM!

File: account_account_tag_code/views/account_account_tag_views.xml [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+        </field>
+    </record>
+
+    <record id="account_tag_view_tree_code" model="ir.ui.view">
+        <field name="name">account.tag.view.tree.code</field>
+        <field name="model">account.account.tag</field>
+        <field name="inherit_id" ref="account.account_tag_view_tree" />
+        <field name="arch" type="xml">
+            <field name="name" position="after">
+                <field name="code" optional="hide" />
```
Comment (AaronHForgeFlow) [inline account_account_tag_code/views/account_account_tag_views.xml]: I would show the field by default. It is an important and generic field.

**Review by AaronHForgeFlow — APPROVED** [review]
LGTM

**Review by sergiocorato — APPROVED** [review]
LGTM

**Review by JordiMForgeFlow — APPROVED** [review]

**Pull request merged**